src/app/components/search/search.component.css

```diff
@@ -2,3 +2,72 @@
   margin-left: 15px;
   padding-left: 30px;
 }
+
```

Comment:
Should all this CSS be in here? Or in a global place? What if we want to use date pickers else where?